RELEASE.md

```diff
@@ -7,20 +7,26 @@ Please follow the established format:
 -->
 # Next release
 
+# Release 9.1.0
+
 ## Major features and improvements
+- Include expandAllPipelines in initial state. (#1896)
+- Allow Kedro-Viz commands to run from any sub directory within Kedro project. (#1871)
 
 ## Bug fixes and other changes
-
-- Upgrade the gitpod workspace-full to a newer version which includes both Node 18 and Python 3.11.5. (#1862)
+- Fix broken URL when active pipeline name changes on initial load. (#1914)
+- Fix bug related to tag filtering and sharing with stateful URL. (#1878)
+- Update settings panel design (#1875)
+- Migrate from CircleCi to Github Actions. (#1876)
 - Refactor backend integration with Kedro by replacing bootstrap_project with configure_project. (#1796)
 - Enhance kedro-viz doc integration. (#1874)
-- Fix Kedro-Viz waiting for valid Kedro project. (#1871)
-- Include expandAllPipelines in initial state. (#1896)
 - Enhance Kedro-Viz documentation by using Kedro-sphinx-theme. (#1898)
 - Remove default props from functional components. (#1906)
```

Comment:
- Include expandAllPipelines in initial state. (#1896)

Reply:
done